snowshu/core/graph_set_runner.py

```diff
@@ -27,7 +28,7 @@
 class GraphExecutable:
     graph: nx.Graph
     source_adapter: BaseSourceAdapter
-    target_adapter: BaseTargetAdapter
+    target_adapter: Union[BaseLocalTargetAdapter, BaseRemoteTargetAdapter]
```

Comment:
Did this one actually need to change?

Reply:
good point, it doesn't I think I wanted to be too specific